package practiceHome;

import java.io.FileNotFoundException;

public class ExceptionHandling {

	public static void main(String[] args) {
//		String s = "400";
//		int a =Integer.parseInt(s);
//		System.out.println(a+ 100);
		//String s = "4.99";
		//double d = Double.parseDouble(s);
		//System.out.println(d+ 20);
		String s = "Monowara Sultana";
		
		try {
			//question :can we have more than one catch block.
			//ans:Yes we have multiple catch block.
			
		}catch(NullPointerException e) {
			e.printStackTrace();	 
			
		}catch(ArithmeticException a) {
			
		}catch(ArrayIndexOutOfBoundsException b) {
			
		}
				
			
			
			
		
		
	
		
		
		

	}

}
